arviz/plots/dotplot.py

```diff
@@ -0,0 +1,229 @@
+"""Plot distribution as dot plot or quantile dot plot."""
+import numpy as np
+
+from ..rcparams import rcParams
+from .plot_utils import get_plotting_function
+
+
+def plot_dot(
+    values,
+    binwidth=None,
+    dotsize=1,
+    stackratio=1,
+    hdi_prob=None,
+    rotated=False,
+    dotcolor="C0",
+    intervalcolor="C3",
+    markersize=None,
+    markercolor="C0",
+    marker="o",
+    figsize=None,
+    linewidth=None,
+    point_estimate="auto",
+    nquantiles=50,
+    quartiles=True,
+    point_interval=None,
+    ax=None,
+    show=None,
+    plot_kwargs=None,
+    backend=None,
+    backend_kwargs=None,
+    **kwargs
+):
+    """Plot distribution as dot plot or quantile dot plot.
+
+    This function uses the Wilkinson's Algorithm[1] to allot dots to bins.
+    The quantile dot plots was inspired from the paper[2].
```

Comment:
```suggestion
    This function uses the Wilkinson's Algorithm [1]_ to allot dots to bins.
    The quantile dot plots was inspired from the paper [2]_.
```

The final underscore i necessary to make those clickable links.